java/arcs/core/analysis/InformationFlow.kt

```diff
@@ -424,6 +438,20 @@ private fun Predicate.labels(): List<InformationFlowLabel> = when (this) {
     is Predicate.And -> lhs.labels() + rhs.labels()
 }
 
+/** Returns true if the [HandleConnectionSpec] is a write. */
+private fun HandleConnectionSpec.isWrite() = when (direction) {
```

Comment:
lol, this should be some kind of bitset-like structure...

Reply:
HandleMode has a `canWrite` property, which what I should use: 
https://github.com/PolymerLabs/arcs/blob/169c4c7fd79f8179b5bff9795fa791e671a5c06c/java/arcs/core/data/HandleMode.kt#L18

I will clean this up in a follow up PR.